scripts/utils/trading_strategy_helpers.js

```diff
@@ -18,67 +28,11 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
   const gnosisSafeMasterCopyPromise = GnosisSafe.deployed()
   const fleetFactoryPromise = FleetFactory.deployed()
 
-  /**
-   * Ethereum addresses are composed of the prefix "0x", a common identifier for hexadecimal,
-   * concatenated with the rightmost 20 bytes of the Keccak-256 hash (big endian) of the ECDSA public key
-   * (cf. https://en.wikipedia.org/wiki/Ethereum#Addresses)
-   * @typedef Address
-   */
-
-  /**
-   * Smart contracts are high-level programming abstractions that are compiled down
-   * to EVM bytecode and deployed to the Ethereum blockchain for execution.
-   * This particular type is that of a JS object representing the Smart contract ABI.
-   * (cf. https://en.wikipedia.org/wiki/Ethereum#Smart_contracts)
-   * @typedef SmartContract
-   */
-
-  /**
-   * Example:
-   * {
-   *   amount: 100,
-   *   tokenAddress: 0x0000000000000000000000000000000000000000,
-   *   bracketAddress: 0x0000000000000000000000000000000000000001
-   * }
-   * @typedef Deposit
-   * @type {object}
-   * @property {integer} amount integer denoting amount to be deposited
-   * @property {Address} tokenAddress {@link Address} of token to be deposited
-   * @property {Address} bracketAddress address of bracket into which to deposit
-   */
-
-  /**
-   * @typedef Withdrawal
-   *  * Example:
-   * {
-   *   amount: "100",
-   *   bracketAddress: "0x0000000000000000000000000000000000000000",
-   *   tokenAddress: "0x0000000000000000000000000000000000000000",
-   * }
-   * @type {object}
-   * @property {integer} amount Integer denoting amount to be deposited
-   * @property {Address} bracketAddress Ethereum address of the bracket from which to withdraw
-   * @property {Address} tokenAddresses List of tokens that the traded wishes to withdraw
-   */
-
-  /**
-   * @typedef TokenObject
-   *  * Example:
-   * {
-   *   symbol: "WETH",
-   *   decimals: 18,
-   *   tokenAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-   *   instance: [object Object],
-   * }
-   * @type {object}
-   * @property {string} symbol symbol representing the token
-   * @property {(number|BN)} decimals number of decimals of the token
-   * @property {Address} address address of the token contract on the EVM
-   * @property {object} instance an instance of the token contract
-   */
-
   /**
    * Returns an instance of the exchange contract
+   *
+   * @param {object} web3 an instance of Web3
+   * @returns {SmartContract} An instance of BatchExchange
    */
   const getExchange = function (web3) {
```

Comment:
`web3` should not be here since it's already defined as the input to the module function! I'd say to remove it in this PR.
```suggestion
  /**
   * Returns an instance of the exchange contract
   *
   * @returns {SmartContract} An instance of BatchExchange
   */
  const getExchange = function () {
```